gulp/DevelopmentTasks.ts

```diff
@@ -158,7 +165,15 @@ const doc: ITaskFunction = async () => {
                 continue;
             }
             totalCommands++;
-            const childAnchorTag = "command-" + child.name.replace(/\s/g, "-");
+
+            // create anchor tag that matches full command
+            var childAnchorTag = ""
```

Comment:
Could you use `let` here as well and a `;`

Reply:
Done.